spec/suma/commerce/checkout_spec.rb

```diff
@@ -345,10 +347,18 @@ def create_order(amount=nil, checkout_: checkout)
         # $20 charge = $15 match voucher cash cost + $5 into voucher cash cost
         order = create_order(money("$20"), checkout_: checkout)
         expect(order.charges).to contain_exactly(have_attributes(discounted_subtotal: cost("$54")))
+        # Credits
+        expect(intro_ledger.received_book_transactions).to contain_exactly(
```

Comment:
There isn't a reason to test this- it's the same set of transactions. There's no way this can fail independently of the other assertions on charges.